packages/batcher/runtime/src/index.ts

```diff
@@ -189,4 +189,7 @@ async function main(): Promise<void> {
   await runtime.run(gameInputValidator, batchedTransactionPoster, provider);
 }
 
-void main();
+main().catch(e => {
+  console.error(e);
```

Comment:
At the least I would add a comment like this

```suggestion
  // we catch the error here instead of relying on `uncaughtException` monitoring
  // this is because if an exception causes us to reach this line, it means the error was thrown during initalization
  // which is typically something we cannot recover from
  console.error(e);
```

If we want to make this a bit more obvious, we could split up the `main()` function into 2 parts:
1. An initialize function
2. The last 2 lines (`requestStart` and `runtime.run`)

then we could add the try/catch just on the `initialize` function

I'm okay with either approach